components/message.rst

```diff
@@ -0,0 +1,284 @@
+.. index::
+   single: Message
+   single: Components; Message
+
+The Message Component
+=====================
+
+    The Message component helps application to send and receive messages
+    to/from other applications or via
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/message
```

Comment:
This should be `symfony/messenger` conform the [composer.json](https://github.com/symfony/messenger/blob/master/composer.json).